packages/zosfiles/src/api/methods/create/Create.ts

```diff
@@ -70,6 +74,10 @@ export class Create {
             case CreateDataSetTypeEnum.DATA_SET_CLASSIC:
                 tempOptions = {...CreateDefaults.DATA_SET.CLASSIC, ...tempOptions};
                 break;
+            // tslint:disable-next-line:no-magic-numbers
```

Comment:
Please do not use magic-numbers.
If this needs to be a constant, then add it to the corresponding constants file and document it.

Reply:
Fixed in recent push :)